setup.cfg

```diff
@@ -1,6 +1,12 @@
+[aliases]
+test=pytest
+
 [metadata]
 description-file = README.rst
 
 [flake8]
 ignore=E402,H238
 max-line-length=99
+
+[tool:pytest]
+[pytest-watch]
```

Comment:
Does this replace nosewatch?

Reply:
not replace but I couldn't get nosewatch working correctly